src/main/resources/generator/android/res/values/styles.xml

```diff
@@ -1,10 +1,5 @@
 <resources>
-    <style name="GdxTheme" parent="android:Theme">
-    <item name="android:windowBackground">@android:color/transparent</item>
-        <item name="android:colorBackgroundCacheHint">@null</item>
-        <item name="android:windowAnimationStyle">@android:style/Animation</item>
-        <item name="android:windowNoTitle">true</item>
-        <item name="android:windowContentOverlay">@null</item>
-        <item name="android:windowFullscreen">true</item>
+    <style name="GdxTheme" parent="android:Theme.Holo.Light.NoActionBar.Fullscreen">
```

Comment:
I don't really know what the difference is between Holo and Material, but I'm hoping that this using Holo is OK... If it isn't, let me know and I can easily swap it out myself (no PR needed).

Reply:
well, Meterial is newer than Holo and looks better but we keep it for pre API 21 devices.